source/import-export.txt

```diff
@@ -49,20 +49,30 @@ that it is formatted correctly.
        name: JSON
        content: |
 
-         When importing data from a **JSON** file, each document must exist on
-         its own line in the file. Do not use commas at the end of lines to
-         separate documents.
+         When importing data from a **JSON** file, you can format your 
+         data as:
+
+         - Newline-delimited documents, or
+
+         - Comma-separated documents in an array
 
          .. example::
 
-            The following ``.json`` file imports three documents:
+            The following newline-delimited ``.json`` file is formatted 
+            correctly:
 
             .. code-block:: javascript
 
                { "type": "home", "number": "212-555-1234" }
                { "type": "cell", "number": "646-555-4567" }
                { "type": "office", "number": "202-555-0182"}
 
+            The following comma-separated ``.json`` array file is also formatted correctly:
+
+            .. code-block:: javascript
+
+               [{ "type": "home", "number": "212-555-1234" }, { "type": "cell", "number": "646-555-4567" }, { "type": "office", "number": "202-555-0182"}]
```

Comment:
Does this format only work if all of the items are on the same line?

Reply:
No, it will ignore line breaks. You could format like this as well:
[
   { ...someJSON },
   { ...so on },
]
I put the JSON example array on the same line to illustrate the difference between formats. I'll add a note that the JSON array format will ignore line breaks, but am open to any format for this. Let me know if you think it's better to show on separate lines.